content/sections.yaml

```
@@ -90,6 +90,14 @@
         color: purple
         repository: wazo-websocketd
 
+      setup:
+        title: Initial system setup
```

Capital letters in title.

```suggestion
        title: Initial System Setup
```